NAMESPACE

```diff
@@ -30,12 +31,17 @@ export(lm_robust_fit)
 export(permutations_to_condition_pr_mat)
 export(tidy)
 importFrom(Rcpp,evalCpp)
+importFrom(stats,.checkMFClasses)
```

Comment:
Be careful about import dotted things, that may trip a cran warning.

Reply:
It doesn't warn with `R CMD check`, and not including it leaves a warning (or a note, I forget which). So I guess we should leave it for now?